### DISCUSSION TITLE
How can I read a very big json file and replace a key value in it. And save it to the original json file.

### DISCUSSION THREAD

**Discussion by phplarvl** [halaxa/json-machine#72] [Q&A]
How can I read a very big json file and replace a key value in it. And save it to the original json file.

**Comment by halaxa** [comment]
Hi. This library has no support for in-place editing. You must read the file sequentially, one item at a time, change (or not) the contents of an item, `json_encode` it and write into another file. You can replace the old file with the new one if you need the changes in the original one.
